health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/ViewHierarchy.js

```
@@ -304,13 +319,18 @@ const ViewHierarchy = () => {
               });
       
               const status = searchResponse?.ResourceDetails?.[0]?.status;
+              let errorString = searchResponse?.ResourceDetails?.[0]?.additionalDetails.error;
```

Biome

[error] This let declares a variable that is only assigned once.

'errorString' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)